libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/SalesforceActivity.java

```diff
@@ -57,7 +57,7 @@ public void onResume() {
 
     @Override
     public void onPause() {
-        super.onPause();
+		super.onPause();
```

Comment:
NT:  Is this an intended change?  (hard to tell what spacing actually is on github)